src/services/rename.ts

```diff
@@ -93,6 +93,7 @@ namespace ts.Rename {
         switch (node.kind) {
             case SyntaxKind.Identifier:
             case SyntaxKind.StringLiteral:
+            case SyntaxKind.NoSubstitutionTemplateLiteral:

```

Comment:
Not sure if there should be more strict checks similar to `NumericLiteral`. I haven't found cases where it would allow improper renames.
What do you think?

Reply:
I think this is right; no reason it shouldn’t follow the same path as StringLiteral 👍

Reply:
Thanks!